src/coreclr/build-runtime.sh

```diff
@@ -250,6 +280,14 @@ __CMakeArgs="-DCLR_CMAKE_PGO_INSTRUMENT=$__PgoInstrument -DCLR_CMAKE_OPTDATA_PAT
 __CMakeArgs="-DCLR_CMAKE_BUILD_SUBSET_JIT=$__BuildJit -DCLR_CMAKE_BUILD_SUBSET_ALLJITS=$__BuildAllJits -DCLR_CMAKE_BUILD_SUBSET_RUNTIME=$__BuildRuntime $__CMakeArgs"
 __CMakeArgs="-DCLR_CMAKE_BUILD_TESTS=$__BuildPALTests $__CMakeArgs"
 
+__CMakeArgs="-DCLI_CMAKE_HOST_VER=$__host_ver -DCLI_CMAKE_COMMON_HOST_VER=$__apphost_ver -DCLI_CMAKE_HOST_FXR_VER=$__fxr_ver $__CMakeArgs"
+__CMakeArgs="-DCLI_CMAKE_HOST_POLICY_VER=$__policy_ver -DCLI_CMAKE_PKG_RID=$__DistroRid -DCLI_CMAKE_COMMIT_HASH=$__commit_hash $__CMakeArgs"
+__CMakeArgs="-DFEATURE_DISTRO_AGNOSTIC_SSL=$__PortableBuild $__CMakeArgs"
+
+if [[ "$__PortableBuild" == 1 ]]; then
+    __CMakeArgs="-DCLI_CMAKE_PORTABLE_BUILD=1 $__CMakeArgs"
```

Comment:
nit: `CLI_CMAKE_PORTABLE_BUILD` is used in one place to define `FEATURE_PORTABLE_BUILD`:
https://github.com/dotnet/runtime/blob/ddf94a71c7606aa433f4776841afd354a33b0502/src/native/corehost/setup.cmake#L4-L6
but according to `git log -SFEATURE_PORTABLE_BUILD` results, `FEATURE_PORTABLE_BUILD` was never used (since it was introduced https://github.com/dotnet/runtime/commit/4f68af44197d0631e3f45673873dd6a69798a1e8). We can delete them both.

Reply:
I think `__PortableBuild` is entirely uninteresting here. 
It basically controls how SSL builds in libraries by setting `FEATURE_DISTRO_AGNOSTIC_SSL`, but that is also done in `build-commons.sh`

Reply:
Yup. It is unused in corehost's Unix shell scripts (because distro specific RID for source-build is taken care by `build-commons.sh`), and for Windows, I think for a while now, nobody is building or testing non-portable flavors.  cm_BaseRid is always overwritten on line 151 https://github.com/dotnet/runtime/blob/39afd1f46f8f03582b0414591ff7a1e2f1809979/src/native/corehost/build.cmd#L146-L151 the default is `true` https://github.com/dotnet/runtime/blob/39afd1f46f8f03582b0414591ff7a1e2f1809979/Directory.Build.props#L95-L96 and CI's YAML configs also set it to true (to make sure it is really ture 🙂).